wdl/gwas/regenie_sub.wdl

```diff
@@ -116,7 +116,34 @@ task step2 {
               continue
             fi
 
-            for s in males females;
+            # NOTE
+            # NOTE figure out whether we can run males and or females sex-specific analysis. In some endpoints, there are too little 
+            # NOTE cases (under 10) per sex, which errors out the script.
+
+            # save a mock file for the case where the analysis could not be performed
+            echo -n "ID A1FREQ_CASES A1FREQ_CONTROLS N BETA SE LOG10P" |bgzip > mock_sex_spec.txt.gz
+            echo '''BEGIN{print_males ="males";print_females="females";males[0]=0;males[1]=0;females[0]=0;females[1]=0}
```

Comment:
working too hard here. No results will be shown in the final sex spec file as we do inner join in python etc. Just test if n cases is lower than males or females and add that into the first if if [[ "${run_sex_specific}" == "true" ]] and then the NOT_DONE file will be created in the else clause.